src/components/button/button.tsx

```diff
@@ -8,17 +8,22 @@ interface IButton {
 
 // Default Button Styles
 const Button = styled.button<IButton>`
-  height: 3rem;
-  min-width: 10rem;
-  width: ${props => (props.screen === "small" ? "70%" : "15rem")};
+  height: 4.5rem;
+  min-width: 8rem;
+  width: ${props => (props.screen === "small" ? "50%" : "15rem")};
   border: none;
+  outline:none;
   background: ${props =>
     props.gradientColor === "purple"
       ? props.theme.gradient.purple
       : props.theme.gradient.green};
   color: ${props => props.theme.colors.pureWhite};
-  border-radius: 0.25rem;
-  font-size: 0.9rem;
+  display:flex;
```

Comment:
What is the purpose of putting flex row in buttons? Are you planning to add icons or more than one child inside of it?

Reply:
I actually added flex when I was testing these out👇 since it was distorted.
```
vertical-align:middle;
text-align:center;
```
Which didn't worked. So tried using flex.
But yes removing that works fine as of now.
```suggestion
  display:flex;     //remove this line and the next 3 lines as well.
```
 justify-content:center;
  align-items:center;
  flex-flow:row wrap;

Reply:
yes remove these flex related properties